admin_guide/high_availability.adoc

```diff
@@ -252,16 +261,71 @@ ip addr show $interface | grep -i MULTICAST
 echo "Check multicast groups ... "
 ip maddr show $interface | grep 224.0.0 | grep $interface
 
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
 EOF
 done;
 ----
 ====
 
+[[options-environment-variables]]
+=== Command Line Options and Environment Variables
+
+.Command Line Options and Environment Variables
+[cols="1a,3a,1a,4a",options="header"]
+|===
+
+| Option | Variable Name | Default | Notes
+
+|--watch-port
+|`*OPENSHIFT_HA_MONITOR_PORT*`
+|80
+|This port must be open for the service to be running.
+
+|--interface
+|`*OPENSHIFT_HA_NETWORK_INTERFACE*`
+|
+|By default all NICs are watched.
```

Comment:
"The interface name for the ip failover to use to send VRRP traffic.  By default, all interfaces are used."